internal/providers/terraform/google/cloudfunctions_function.go

```diff
@@ -0,0 +1,135 @@
+package google
+
+import (
+	"github.com/infracost/infracost/internal/schema"
+	"github.com/shopspring/decimal"
+)
+
+func GetCloudFunctionsRegistryItem() *schema.RegistryItem {
+	return &schema.RegistryItem{
+		Name:  "google_cloudfunctions_function",
+		RFunc: NewCloudFunctions,
+	}
+}
+
+func NewCloudFunctions(d *schema.ResourceData, u *schema.UsageData) *schema.Resource {
+	region := d.Get("region").String()
+
+	memorySize := decimal.NewFromInt(256)
+	if d.Get("available_memory_mb").Exists() {
+		memorySize = decimal.NewFromInt(d.Get("available_memory_mb").Int())
+	}
+
+	var cpuMapping = map[int]decimal.Decimal{
+		128:  decimal.NewFromInt(200),
+		256:  decimal.NewFromInt(400),
+		512:  decimal.NewFromInt(800),
+		1024: decimal.NewFromInt(1400),
+		2048: decimal.NewFromInt(2400),
+		4096: decimal.NewFromInt(4800),
+	}
+
+	cpuSize := cpuMapping[int(memorySize.IntPart())]
+
+	invocations := decimal.Zero
+	if u != nil && u.Get("monthly_function_invocations").Exists() {
+		invocations = decimal.NewFromInt(u.Get("monthly_function_invocations").Int())
+	}
+
+	var networkEgrees *decimal.Decimal
+	if u != nil && u.Get("monthly_outbound_data_gb").Exists() {
+		networkEgrees = decimalPtr(decimal.NewFromInt(u.Get("monthly_outbound_data_gb").Int()))
+	}
+
+	requestDuration := decimal.Zero
```

Comment:
The pricing page mentions "compute time is measured in 100ms increments, rounded up to the nearest increment" So I think we should:
1. Default this to 100
2. Inside the if-statement, round **up** the user input to the nearest 100ms, e.g. if they provide 240, we set it to 300.